### PR TITLE
Support Speedup for Slim Pruner.

### PR DESCRIPTION
**Pull request by zheng-ningxin**
Speedup does not support Slim pruners at the beginning(v1.6). Slim pruner prunes the BN layers which is not supported in the
mask_conflict utils, in this PR, we support the BN layers in mask_conflict so that Slim pruner is supported in speedup.

### CODE REVIEW

File: nni/compression/pytorch/utils/shape_dependency.py [hunk 1]
```diff
@@ -98,7 +98,18 @@ def __init__(self, model=None, dummy_input=None, traced_model=None):
         traced_model : torch._C.Graph
             if we alreay has the traced graph of the target model, we donnot
             need to trace the model again.
-        """
+        prune_type: str
+            This parameter indicates the channel pruning type: 1) `Filter`
+            prune the filter of the convolution layer to prune the corresponding
+            channels 2) prune the channel in the batchnorm layer
```
Comment (J-shang) [inline nni/compression/pytorch/utils/shape_dependency.py:104]: 2 ) `Batchnorm` prune ...

**Review by J-shang — APPROVED** [review]

File: nni/compression/pytorch/utils/mask_conflict.py [hunk 2]
```diff
@@ -307,10 +309,43 @@ def fix_mask(self):
...
+    followed after a conv(Slim pruner). This function find
+    the pruning type of the masks.
+
+    Parameters
+    ----------
+    masks: dict
+        A dict object that stores the masks.
+    model: nn.Module
+        Model object which the mask can be applied on.
+    Returns:
```
Comment (QuanluZhang) [inline nni/compression/pytorch/utils/mask_conflict.py:326]: add blank line

**Review by QuanluZhang — APPROVED** [review]
